test-week.js

```diff
@@ -0,0 +1,748 @@
+#!/usr/bin/env nodejs
```

Comment:
You can remove the file directly. It can be left in git history. The tests are already in the main test file.